src/main/resources/META-INF/rewrite/jakarta-ee-9.yml

```diff
@@ -192,6 +198,12 @@ recipeList:
       newGroupId: jakarta.authentication
       newArtifactId: jakarta.authentication-api
       newVersion: latest.release
+  - org.openrewrite.java.dependencies.ChangeDependency:
+      oldGroupId: org.glassfish.main
+      oldArtifactId: javax.security.auth.message
+      newGroupId: jakarta.authentication
+      newArtifactId: jakarta.authentication-api
+      newVersion: latest.release
```

Comment:
If I follow the relocation meta data I arrive at slightly different artifact
https://mvnrepository.com/artifact/org.glassfish.main/javax.security.auth.message
```suggestion
  - org.openrewrite.java.dependencies.ChangeDependency:
      oldGroupId: org.glassfish.main
      oldArtifactId: javax.security.auth.message
      newGroupId: jakarta.security.auth.message
      newArtifactId: jakarta.security.auth.message-api
      newVersion: latest.release
```
Note that I know nothing of the internals, so feel free to correct me. :)

Reply:
Just had a closer look; does appear as though `jakarta.authentication-api` contains a jar with package structure that appears to match. Let's keep this one as-is then.
![image](https://github.com/user-attachments/assets/505e28b8-ee80-4828-b4e7-bbc1dcae0655)